packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -1,145 +1,43 @@
 import React, { FC } from 'react';
 import {
-  useParams,
-  Column,
   TableProvider,
   createTableStore,
-  useDocument,
-  useToggle,
-  useOmSupplyApi,
   Item,
+  useEditModal,
```

Comment:
This gets changed to be similar to all the other detail view pages - just uses `useEditModal` and the modal itself handles all of the things